doc/api/repl.markdown

```diff
@@ -64,6 +64,9 @@ the following values:
    returns the formatting (including coloring) to display. Defaults to
    `util.inspect`.
 
+ - `completer` - an optional function that is used for Tab autocompletion.
+   See [Readline Interface][] for an example of using this.
```

Comment:
Well, at the time I made the PR, that was the way in the other docs that the linking was done, maybe this changed in the meantime. Search in https://raw.githubusercontent.com/joyent/node/master/doc/api/https.markdown for `[tls.createServer()][]` for an example.

If there is other way to do it, just point me in the rigth direction please, but looking at the docs code I didn't found an example of direct linking.